### PR TITLE
Add a character sheet

### PR DESCRIPTION
**Pull request by agude**
Here is a first draft of a character sheet. I think having this will give us some ideas about what still needs work (like... how do you advance? Or make more powerful characters?).

I don't **LOVE** everything, like I think the `<h3>` elements waste too much space, but it's a good start.

### Unfilled

image

### Filled

image

### CODE REVIEW

**Comment by agude** [comment]
The power of *GRID*:

image

File: assets/_sass/main.scss [hunk 1]
```diff
@@ -260,3 +264,174 @@ ul > li:before {
...
+  "name name name"
+  "backstory backstory backstory"
+  "aspect aspect fatepoints"
+  "stressbox consequence consequence";
+  justify-content: stretch;
+  align-content: stretch;    
+}
+
+div.character-sheet-section {
+  //justify-self: start;
```
Comment (charles-uno) [inline assets/_sass/main.scss:287]: delete?

**Comment by charles-uno** [comment]
Looks like a great start!

Not wild about having Concept and Trouble show up twice. Maybe lose the gray text and put the prompt underneath or to the side?

**Review by charles-uno — APPROVED** [review]

**Comment by agude** [comment]
Placeholder text gone.

Screenshot 2023-03-03 at 7 47 12 PM

**Pull request merged**